src/appShell/App/Container.tsx

```diff
@@ -62,16 +63,29 @@ export default class Container extends React.Component<IContainerProps, {}> {
                             <PortalHeader appStore={this.appStore}/>
                         </div>
                     </div>
-                    <If condition={this.appStore.isErrorCondition}>
-                       <Then>
-                           <div className="contentWrapper">
-                               <ErrorScreen
-                                    title={"Oops. There was an error retrieving data."}
-                                    body={<a href={buildCBioPortalPageUrl("/")}>Return to homepage</a>}
-                                    errorLog={formatError(this.appStore.undismissedSiteErrors)}
-                               />
-                           </div>
-                       </Then>
+                    <If condition={this.appStore.isErrorCondition || !ServerConfigHelpers.sessionServiceIsEnabled()}>
```

Comment:
You'll notice above that we check for isSessionLoaded.   Apparently that will evaluate as true if session service is missing.  Nevertheless, for clarity, I think we should put this new ErrorScreen outside that check.  The render method can just return <ErrorScreen> component and nothing else.

Reply:
In other words, at line 49.   sessionServiceDisabled.

Reply:
updated